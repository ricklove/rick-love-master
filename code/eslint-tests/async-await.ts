export const requireAwait = async () => { return true; };