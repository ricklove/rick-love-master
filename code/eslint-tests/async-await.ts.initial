export const requireAwaitExists = async () => { return true; };

export const basicPromise = () => new Promise<boolean>((resolve) => { resolve(true); });
export const requireAwaitOnPromise = async () => { return basicPromise(); };