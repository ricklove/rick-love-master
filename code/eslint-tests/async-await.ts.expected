export const requireAwaitExists = async () => { return await Promise.resolve(true); };

export const basicPromise = () => new Promise<boolean>((resolve) => { resolve(true); });
export const requireAwaitOnPromise = async () => { return await basicPromise(); };
